tests/unit/feature_extractors/test_deepwalk.py

```diff
@@ -1,163 +0,0 @@
-# Copyright © 2022-2022 Blue Brain Project/EPFL
```

Comment:
We are removing all unit tests for the submodules of `morphoclass.feature_extractors` that were removed in this PR, i.e. `interneurons, pyramidal_cells, feature_extraction`.

However, we don't have unit tests for the only submodule left, i.e. `morphoclass.features.non_graph`.
It was already a problem before this PR, but just leaving this comment here so we can address it in the future.